sdk/core/Azure.Core/src/Shared/AutoRest/StringRequestContent.cs

```diff
@@ -19,7 +19,7 @@ public StringRequestContent(string value)
 
         public override async Task WriteToAsync(Stream stream, CancellationToken cancellation)
         {
-            await stream.WriteAsync(_bytes, 0, _bytes.Length, cancellation).ConfigureAwait(false);
```

Comment:
We might need to bring this change into the autorest repo. The build would fail without it.

Reply:
Will update this PR with right version after https://github.com/Azure/autorest.csharp/pull/696 is merged.

Reply:
Updated. Should build now.